apps/projects/app/components/Form/Input/DateInput.js

```diff
@@ -8,22 +8,22 @@ import DatePicker from './DatePicker'
 import { IconCalendar } from '../../Shared'
 
 const Container = styled.div`
-  width: 200px;
-  display: flex;
+  width: 100%;
+  position: relative;
 `
 const IconWrapper = styled.div`
-  position: relative;
-  left: -28px;
-  top: 8px;
+  position: absolute;
+  right: 12px;
+  top: 12px;
   height: 14px;
 `
 const TextInputDate = styled(TextInput).attrs({
   readOnly: true
 })`
-  width: 123px;
-  height: 32px;
+  height: 40px;
   display: inline-block;
   padding-top: 3px;
+  width: 100%;
```

Comment:
Not sure if these changes are also useful for rewards apps, in case they are, I think it would be a good idea to put them into a shared folder so we don't need to duplicate efforts.